### ISSUE TITLE
Multiple file support

### ISSUE THREAD

**Issue by circuitsacul** [Lunarmagpie/io#11]
It would be cool if I could upload files as modules (or have seperate codeblocks as modules maybe). So I could do

module.py
```py
def thing():
    print("hi")
```
```py
from module import thing
thing.thing()
```

Basically it should just add the file with the provided filename to the same directory as the main function. This would probably require a piston change though so idk.

**Comment by zunda-arrow** [comment]
Piston supports this but godbolt doesn't. Because there isn't any godbolt support I'm apprehensive to add this feature right now.

**Comment by circuitsacul** [comment]
If a message requesting execution has a feature that isn't supported by one of the providers, just don't list the compilers for that provider I think. I'm happy to implement this

**Comment by zunda-arrow** [comment]
I will merge this if it is implemented

**Comment by MicaelJarniac** [comment]
> If a message requesting execution has a feature that isn't supported by one of the providers, just don't list the compilers for that provider I think. I'm happy to implement this

Not sure if just not listing the unsupported providers would be best. Could end up confusing the user as to why certain providers are missing.

**Comment by circuitsacul** [comment]
Is there a way to disable specific options in a dropdown @Lunarmagpie?

**Comment by zunda-arrow** [comment]
You can filter the options before they are added to the dropdown.

**Comment by MicaelJarniac** [comment]
> Is there a way to disable specific options in a dropdown @Lunarmagpie?

Do you mean like having the option on the dropdown, but sort of grayed out so one can see it but not select it?

**Comment by zunda-arrow** [comment]
> Do you mean like having the option on the dropdown, but sort of grayed out so one can see it but not select it?

I assumed circuit wanted to remove the option from the dropdown. I prefer this than making it grayed out.

**Comment by circuitsacul** [comment]
> Do you mean like having the option on the dropdown, but sort of grayed out so one can see it but not select it?

Yes that's what I meant. IMO It would be less confusing than having a bunch of compilers just disappearing for no apparent reason.
